tests/conftest.py

```diff
@@ -18,14 +18,13 @@
 
 import pytest
 
+from tests.emulator_tests.device_handler import BackgroundDeviceHandler
```

Comment:
Hm, what is wrong with that? I mean we use the device_handler atm only for `emulator_tests`, right? So it makes sense to have it there. But I guess I can leave it in the root as well, it just seems a bit messy already (there is ~10 files already there which are not very related).

Reply:
Having the `device_handler` fixture defined in root conftest makes it available for all tests. So it "feels wrong" to have the implementation scoped in emulator tests.

A possible solution would be to scope the fixture itself, but I'm not sure if that's possible. Last time I tried, multiple `conftest`s didn't work well together.

Reply:
> A possible solution would be to scope the fixture itself, but I'm not sure if that's possible. Last time I tried, multiple conftests didn't work well together.

Nuh, I don't like that myself.

Ok, so I'll move `device_handler` to the root, but it makes sense to leave `emulators.py`, doesn't it?

Reply:
for now, perhaps. but in the near future we will need `emulators` (which should be called `emulator` probably) in root, so that the test suite can run an emulator automatically for device tests as well

Reply:
Ok, let's do it like that for now.

Reply:
Fixed and squsahed.